### ISSUE TITLE
New Reticle and Indoor Ambient

### ISSUE THREAD

**Issue by vitorbalbio** [googlevr/gvr-unity-sdk#178]
Hi!
The new Reticle system released with 0.6 is very nice but it don't have anymore (or i don't get how to use) a important feature for indoor ambient. Since it have fixed position at a some distance if you have a object close to you that is not a target you get a huge eye strain that mess the experience. The old pointer system has a option to update the position and put it always on the objects. Even the non interactive ones.
Any tips on it?

**Comment by smdol** [comment]
Hmm.  Good point.  The distance function should not necessarily be tied to the is-clickable function.  I can get back to you with a workaround.

**Comment by vitorbalbio** [comment]
Hi smdol, it would be great sure it would help me a lot!
Thanks!

**Comment by vitorbalbio** [comment]
I made a patch to "fix" this limitation. What i did was add support to set the layers that trigger the animation. That way we can have the different situations: 

- Object is a target ( It layer is in the Layermask of the Raycaster so it update the reticle position and we can send events to it ) and is also a trigger to the reticle animation ( So it layer also is in the layermask of the mostly used for interactive ones ). 

- Object is a target but not trigger the reticle animation ( So it layer is not in the Reticle Anim layermask), Used for all other Solid but non interactive objects of the scene.

- Objects that are no target at all.

I tried to make a Pull request but mess with reverts, i will try do it right now.
Bye!
